Installation/CHANGES.md

```diff
@@ -1,6 +1,23 @@
 Release History
 ===============
 
+Release 4.13
+------------
+
+Release date: September 2018
+
+### Polygon Mesh Processing
+
+-  Fix a bug in `isotropic_remeshing()` making constrained vertices missing in the output
+
+### CGAL and the Boost Graph Library (BGL)
+
+-   Update the function `CGAL::Euler::collapse_edge` so that the target
+    vertex of the edge collapsed is always kept after the collapse.
```

Comment:
- I would replace "Update" with "Improve"
- "edge collapsed" should better be "collapsed edge", imho